rest_api/api/assets.py

```diff
@@ -62,8 +64,9 @@
 @ASSETS_BP.get('assets/<name>')
 async def get_asset(request, name):
     """Fetches the details of particular Asset in state"""
+    decoded_name = unquote(name)
```

Comment:
Seems to me this commit could be squashed with the original commit that added this code.